Makefile.am

```diff
@@ -208,6 +208,7 @@ LCOV_FILTER_PATTERN = \
 	-p "src/univalue" \
 	-p "src/crypto/ctaes" \
 	-p "src/secp256k1" \
+	-p "src/simplicity/secp256k1" \
```

Comment:
I'm confused why this isn't just `"src/simplicity"`.   I'm also confused as to why you got coverage output for all the simplicity files, even tough it wasn't included in this filter.

Reply:
Oh the filter "removes" these directories.  Got it.